lib/game_box_web/router.ex

```diff
@@ -39,6 +39,7 @@ defmodule GameBoxWeb.Router do
 
     live_session(:default, on_mount: GameBoxWeb.InitAssigns) do
       live("/", HomeLive)
+      live("/join", HomeLive)
```

Comment:
do we need to use the `/join` route or would the same logic work for the root with params?

Reply:
Either would work, but I was thinking we're about to have to paths for "start a game" vs "join a game" because you're working on the code generator, so it keeps the link from changing in the future.